examples/dependency/src/dependencies-example-index.ts

```diff
@@ -35,6 +36,12 @@ function addDependencyInitializerOnClick() {
         // Change properties of telemetry event "before" it's been processed
         details.item.name = "dependency-name";
         details.item.properties.url = details.item?.target;
+        if (isString(details.item.properties.url)) {
```

Comment:
You don't need to add this code in example apps we have here, we should be redacting the URL internally so this is more confusing than helpful